### DISCUSSION TITLE
Feature Request: Top 7 badge on awarded post

### DISCUSSION THREAD

**Discussion by AneeqaKhan** [forem/forem#17641] [Feature Requests]
We can show a "Top 7 badge" on that related post. (maybe beside its title or maybe on the side bar)
It'll also help to add that article as an achievement on my social media profile.

**Comment by michael-tharrington** [comment]
That would be quite cool! And as a reader, I'd love it if I had a way to filter to all of the top 7 posts, somehow. Would be excellent if we could do both!

**Comment by andypiper** [comment]
I like both of these ideas. Are badges supported across all Forem instances / are they a core part of the platform, or are they (now) specific to DEV?

Adding badges as a search filter for both users and posts would be a nice extra.

**Reply by michael-tharrington** [reply]
Badges are indeed supported across all Forem instances. And wow, it would be cool to have a search filter for badges in some form!
